src/content/general/source-data/materials/readme.md

```diff
@@ -0,0 +1,15 @@
+This page will serve as a hub for the symbols and properties users can assign to their materials. We will be learning how the name of our material in our 3D editing software is used to find a valid shader tag, how various special properties can be assigned to a surface through a specific symbol, and set how the lightmapper will use our surface during rasterization. Some examples are adding the `^` symbol to the end of a material name (like `vines^`) so it becomes a climbable ladder, or the special name `+sky` which causes the marked surface to render our assigned sky.
```

Comment:
I think when most of the content on a topic is written we should not say things like “this page will contain..” it’s very placeholder-ey. This page should take a more definitive tone than conversational because it’s not a guide. The first sentence should explain what the topic is at a high level:


Halo model assets use a special naming convention for their materials which is understood by each game’s Tool when importing those assets into tags.